plugins/quickinspector/quickoverlay.cpp

```diff
@@ -437,17 +441,18 @@ void QuickOverlay::windowAfterRendering()
         m_grabbedFrame.transform.translate(intersect.x() * m_renderInfo.dpr , -intersect.y() * m_renderInfo.dpr - h);
         m_grabbedFrame.image.setDevicePixelRatio(m_renderInfo.dpr);
 
-        if (!m_grabbedFrame.image.isNull()) {
-            m_sceneGrabbed.invoke(this, Qt::QueuedConnection, Q_ARG(GammaRay::GrabbedFrame, m_grabbedFrame));
-        }
+        // Let emit the signal even if our image is possibly null, this way we make perfect ping/pong
+        // reuests making it easier to unit test.
```

Comment:
Yes it can handle null images.
Technically I never had a case of null image, but I suppose it can arrive if the window root item has a size of 0x0 (yeah that's improbable case but still possible).
Does cost nothing as far as I can see.